src/SDL_shadercross.c

```diff
@@ -119,6 +119,14 @@ struct IDxcBlob
     IDxcBlobVtbl *lpVtbl;
 };
 
+static int parse_version_number(const char* str)
+{
+    unsigned major, minor, patch;
+    if(SDL_sscanf(str, "%u.%u.%u", &major, &minor, &patch) == 3)
```

Comment:
Sorry, one last thing for real... space after if and braces on if statements please.

```c
if (...) {
  ...
}